packages/website/src/docs/form/form-libraries.mdx

```diff
@@ -35,6 +35,61 @@ function Form() {
 export default Form;
 ```
 
+An example of a more complete integration leveraging fieldwrappers, formik errors, zod, etc.:
+```jsx
+import React from 'react';
+import { Formik, Form, Field as FormikField } from 'formik';
+import { Button, FieldWrapper, Input } from 'bumbag';
+import { validateZodSchema } from "blitz"
+import { z } from "zod"
+
+export const FormSchema = z.object({
+  username: z.string().nonempty("Provide a valid username."),
+})
+
+function Form() {
+  return (
+    <Formik
+      initialValues={{ username: '' }}
+      onSubmit={data => console.log(data)}
+      validate={validateZodSchema(FormSchema)}
+    >
+      {(formik) => (
+        <Form>
+          <FieldWrapper
+            state={formik.errors.username ? "danger" : undefined}
+            validationText={formik.errors.username && `${formik.errors.username}`}
+          >
+            <FormikField
+              component={Input.Formik}
+              value={formik.values.username}
```

Comment:
Bumbag's `InputField.Formik` component already takes care of this validation. Perhaps remove the `FieldWrapper` in favour of `InputField.Formik`?